pkg/keys/rsa_test.go

```diff
@@ -28,3 +29,16 @@ func (RsaSuite) TestMarshalUnmarshal(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(pubKey.MarshalPublicKey(), DeepEquals, publicData)
 }
+
```

Comment:
Missing a few more tests, no, like signing and verifying? Would be nice to see the tests match the ecdsa ones.